packages/beeq/src/components/page-title/_storybook/bq-page-title.stories.tsx

```diff
@@ -0,0 +1,110 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+import { html, nothing } from 'lit-html';
+
+import mdx from './bq-page-title.mdx';
+
+const meta: Meta = {
+  title: 'Components/Page title',
+  component: 'bq-page-title',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  argTypes: {
+    'have-back-navigation': { control: 'boolean' },
+    customDivider: { control: 'boolean', table: { disable: true } },
+    title: { control: 'text', table: { disable: true } },
+    'sub-title': { control: 'text', table: { disable: true } },
+    actions: { control: 'text', table: { disable: true } },
```

Comment:
Please, try always to make a clear differentiation between the component's props and those used only for testing/documentation purposes.

```suggestion
    // Not part of the component's API, so we don't want to expose it in the controls panel
    customDivider: { control: 'boolean', table: { disable: true } },
    title: { control: 'text', table: { disable: true } },
    'sub-title': { control: 'text', table: { disable: true } },
    actions: { control: 'text', table: { disable: true } },
```